packages/server/tests/acceptance/conformityTests.spec.ts

```diff
@@ -364,3 +365,151 @@ describe('@api-conformity Ethereum execution apis tests', function () {
+    });
+
+    existingContractFilter = (
+      await sendRequestToRelay(
+        {
+          jsonrpc: '2.0',
+          method: 'eth_newFilter',
+          params: [
+            {
+              fromBlock: '0x1',
+              toBlock: '0x160c',
+              address: deployLogsContractTx.contractAddress,
+            },
+          ],
+          id: 1,
+        },
+        false,
+      )
+    ).result;
+  });
+
+  const TEST_CASES = {
+    eth_submitHashrate: {
+      status: 400,
+      request: '{"jsonrpc":"2.0","id":1,"method":"eth_submitHashrate"}',
+      response: '{"jsonrpc":"2.0","id":1,"error":{"code":-32601}}',
+    },
+    eth_sign: {
+      status: 400,
+      request: '{"jsonrpc":"2.0","id":1,"method":"eth_sign"}',
+      response: '{"jsonrpc":"2.0","id":1,"error":{"code":-32601}}',
+    },
+    eth_signTransaction: {
+      status: 400,
+      request: '{"jsonrpc":"2.0","id":1,"method":"eth_signTransaction"}',
+      response: '{"jsonrpc":"2.0","id":1,"error":{"code":-32601}}',
+    },
+    eth_sendTransaction: {
+      status: 400,
+      request: '{"jsonrpc":"2.0","id":1,"method":"eth_sendTransaction"}',
+      response: '{"jsonrpc":"2.0","id":1,"error":{"code":-32601}}',
+    },
+    eth_protocolVersion: {
+      status: 400,
+      request: '{"jsonrpc":"2.0","id":1,"method":"eth_protocolVersion"}',
+      response: '{"jsonrpc":"2.0","id":1,"error":{"code":-32601}}',
+    },
+    eth_newPendingTransactionFilter: {
+      status: 400,
+      request: '{"jsonrpc":"2.0","id":1,"method":"eth_newPendingTransactionFilter"}',
+      response: '{"jsonrpc":"2.0","id":1,"error":{"code":-32601}}',
+    },
+    eth_newBlockFilter: {
+      request: '{"jsonrpc":"2.0","id":1,"method":"eth_newBlockFilter"}',
+      response: '{"jsonrpc":"2.0","id":1,"result":"0x33f5c9d2974eea142909a19906ef0548"}',
+    },
+    'eth_getFilterChanges - existing filter': {
```

Comment:
If you need this to be string for the updateParamIfNeeded, can you just parse to string before you pass it to the function instead of doing it here? It would look "prettier" and more consistent